sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/Exemplar.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.data;
+
+import io.opentelemetry.api.common.Attributes;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * A sample input measurement.
+ *
+ * <p>Exemplars also hold information about the environment when the measurement was recorded, for
+ * example the span and trace ID of the active span when the exemplar was recorded.
+ */
+@Immutable
+public interface Exemplar {
+  /**
+   * The set of key/value pairs that were filtered out by the aggregator, but recorded alongside the
+   * original measurement. Only key/value pairs that were filtered out by the aggregator should be
+   * included
+   */
+  Attributes getFilteredAttributes();
+
+  /** Returns the timestamp in nanos when measurement was collected. */
+  long getRecordTimeNanos();
```

Comment:
How about `getEpochNanos()` to match `PointData`? This is semantically the same thing right?

Reply:
Though renaming all to `getTimestampNanos()` would probably be good IMO as a separate discussion

Reply:
This field matches the OTLP proto field.  I think there's a good pairing between the two as It's easy to see how java is representing OTLP, and how the internal datamodel can be used for exports.  If you'd like to rename, perhaps discussing there would be the best option.  For now, I'd prefer to keep the `metrics.data` package in line with OTLP / OTel metrics data model.

Reply:
Thanks for reminding of OTLP. So yeah `getEpochNanos` is correct I guess since it's how we translate OTLP `time_unix_nanos` for all the other data fields in this SDK I think

Reply:
done!